packages/components/filters/src/badge/badge.tsx

```diff
@@ -1,5 +1,57 @@
-function Badge() {
-  return <div>badge</div>;
+import { css } from '@emotion/react';
+import { designTokens } from '@commercetools-uikit/design-system';
+
+export type TBadgeProps = {
+  /**
+   * CSS ID for badge, used to specify relationship with parent in parent's `aria-controls` property
+   * see https://www.w3.org/TR/wai-aria-1.1/#status
+   */
+  id: string;
+  /**
+   * If `true`, indicates that the element is in a disabled state.
+   */
+  isDisabled?: boolean;
+  /**
+   * String to be displayed in badge, generally a count of some kind
+   */
+  label: string;
+
+  /**
+   * Optional descriptive explanation of label (e.g. "+4 additional filters applied")
+   */
+  ['aria-label']?: string;
+};
+
+const badgeStyles = css`
+  font-size: ${designTokens.fontSize20};
+  font-weight: ${designTokens.fontWeight500};
+  line-height: ${designTokens.lineHeight20};
+  color: ${designTokens.colorSurface};
+  background-color: ${designTokens.colorInfo};
+  padding: 0 calc(${designTokens.spacing05} + ${designTokens.spacing10});
+  border-radius: ${designTokens.borderRadius20};
+  display: inline-flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: ${designTokens.spacing40};
+`;
+
+const disabledBageStyles = css`
+  background-color: ${designTokens.colorNeutral};
+`;
+
+function Badge(props: TBadgeProps) {
+  return (
+    <span
```

Comment:
Here you would use the `StyledBadge` instead of `<span>`. 

Nice to have: Typescript is aware that `StyledBadge` uses a span tag to render to DOM and only suggests attributes that you can set on a span-tag. Granted, not super useful in this case, but very much more so on `button`'s and `input`'s.

Reply:
oh, this is an argument for `styled` I hadn't considered.  personally I think it's weird to couple styling so heavily to markup, but css-in-js pretty much always results in coupling them anyway.

My suggestion here is lets use `css` for this component, and `styled` for the next new component, and then reassess?

Reply:
As mentioned it's just a proposal. I am aware of the current state in ui-kit. It's just the first time we actually add something instead of just fixing something, so I at least wanted to mention it. As for the markup: `styled` also adds the `as` property to the component, so you can change the underlying semantics on a case by case basis.

Reply:
I really more meant that with `css` we could at least theoretically use the `disabled` `css` function in more than one place, kind of like, yknow, css.  But in reality we don't do that enough to optimize for that behavior (although I'm going to argue that maybe we should when it comes to actually standardizing styling practices)